### PR TITLE
remove usages of HtmlView(String,String)

### PR DESCRIPTION
**Pull request by labkey-matthewb**
#### Rationale
Deprecated method needs to be removed

#### Related Pull Requests
* 

#### Changes
*

### CODE REVIEW

File: ms2/src/org/labkey/ms2/protein/tools/GoLoader.java [hunk 1]
```diff
@@ -69,24 +66,25 @@ public abstract class GoLoader
     private static Boolean _goLoaded = null;
     private static GoLoader _currentLoader = null;
 
-    private final StringBuffer _status = new StringBuffer();  // Can't use StringBuilder -- needs to be synchronized
+    private final HtmlStringBuilder _status = HtmlStringBuilder.of();  // Can't use StringBuilder -- needs to be synchronized
```
Comment (labkey-adam) [inline ms2/src/org/labkey/ms2/protein/tools/GoLoader.java:69]: Comment suggests we need to synchronize access to `_status`. Should either add this or remove the comment.

**Review by labkey-adam — CHANGES_REQUESTED** [review]
Address synchronization comment

File: ms2/src/org/labkey/ms2/protein/tools/GoLoader.java (hunk 1 above)
Comment (labkey-matthewb) [reply ms2/src/org/labkey/ms2/protein/tools/GoLoader.java:69]: Hmmm

Comment (labkey-adam) [reply ms2/src/org/labkey/ms2/protein/tools/GoLoader.java:69]: Turns out GO loading is quite broken: https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=49301

Send fine to just merge as is.

**Review by labkey-adam — APPROVED** [review]

**Pull request merged**